otx/algorithms/detection/configs/instance_segmentation/configuration.yaml

```diff
@@ -238,6 +238,21 @@ postprocessing:
     value: 0.01
     visible_in_ui: true
     warning: null
+  use_ellipse_shapes:
+    affects_outcome_of: INFERENCE
+    default_value: false
+    description: Use direct ellipse shape in inference instead of polygon from mask
+    editable: true
+    header: Use ellipse shapes
+    type: BOOLEAN
+    ui_rules:
+      action: DISABLE_EDITING
+      operator: AND
+      rules: []
+      type: UI_RULES
+    value: false
+    visible_in_ui: false
```

Comment:
Specific reason to hide?

Reply:
No specific reason, I just think that too many configurable parameters could make users confused.